### ISSUE TITLE
track coupon code usage in Google Analytics

### ISSUE THREAD

**Issue by linclark** [npm/newww#975]
@seldo requested that this data be added to the data we collect with GA.

**Comment by aredridel** [comment]
@npm/marketing Is this something that is still needed / on your radar?

**Comment by cowperthwait** [comment]
Yes; I would roll that into [Laurie’s CMS superticket](https://github.com/npm/issue-tracker/issues/374). We need to revisit what pre-purchase, purchase, and post-purchase activities (both on-site and in the CLI) get pushed into GA as activities. I would consider specific coupon validation to be a nice-to-have piece of metadata that we would collect _inter al._
